xmtp_mls/src/groups/intents.rs

```diff
@@ -222,6 +224,48 @@ impl From<RemoveMembersIntentData> for Vec<u8> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct UpdateMetadataIntentData {
+    pub group_name: String,
+}
+
+impl UpdateMetadataIntentData {
+    pub fn new(group_name: String) -> Self {
+        Self { group_name }
+    }
+
+    pub(crate) fn to_bytes(&self) -> Vec<u8> {
```

Comment:
Should we just unify all the conversion methods in the `TryFrom` to avoid confusion; what's the benefit of having the method on `UpdateMetadataIntentData`?

Reply:
Updated [here](d91fe5b52db8f62f4aa20fb77cd3a6b87b64829b). I think this is what you were suggesting, but let me know if you meant something else.